### DISCUSSION TITLE
React 18 StrictMode breaks default styling

### DISCUSSION THREAD

**Discussion by axhl** [mantinedev/mantine#128] [Q&A] [answered]
Thanks for the great work that's gone into the library thus far. 

`<React.Strictmode>` in the latest React 18 release currently prevents components from rendering with default styling. Codesandbox to demonstrate: https://codesandbox.io/s/mantine-react18-strictmode-u0s9p

My guess is that this is due to the addition of Strict Effects per https://github.com/reactwg/react-18/discussions/19

Is this a known issue? Or considered an issue at all?

**Comment by rtivital** [comment, marked as answer]
Hi, thanks for reporting, I think that this is most likely a react-jss issue, I'll connect with jss team and debug this once react 18 rc will be out

**Reply by danioo** [reply]
Any updates here?

**Reply by rtivital** [reply]
Yes, we've migrated project to emotion and do not use react-jss anymore

**Reply by rtivital** [reply]
Works fine – https://codesandbox.io/s/mantine-react18-strictmode-forked-rfce9?file=/src/App.tsx
